src/exposition/http.rs

```diff
@@ -107,6 +121,40 @@ mod handlers {
                         }
                     }
                 }
+                if config.general().prometheus_histograms() {
+                    if let Some(snapshot) = snapshots.previous.get(metric.name()) {
+                        // downsample to 6.25% error
+                        let snapshot = snapshot.downsample(3).unwrap();
```

Comment:
Do you want to make the downsampling part of the config?

Reply:
Added to the config. Moved prometheus config to its own section.